api/diagnostics_test.go

```diff
@@ -567,6 +567,7 @@ func TestCancelWhenJobIsRunning(t *testing.T) {
 
 	<-called
 	require.True(t, job.getBundleReportStatus().Running)
+	require.Empty(t, job.getBundleReportStatus().JobEnded, "job is running, end time should be empty")
```

Comment:
Personally, I'm not really a fan of using empty/non-empty strings for boolean logic. Shouldn't we just check if the job is running or not using `job.Running` like you do on the line above?

Reply:
This is test to check if logic above is right

Reply:
Ah, makes sense.